### PR TITLE
Update LICENSE files and license headers for python sdk

### PR DESCRIPTION
**Pull request by amulyavarote**
# Description

Per CNCF requirements, replacing MIT License with Apache.

## Issue reference

We strive to have all PR being opened based on an issue, where the problem or feature have been discussed prior to implementation.

Please reference the issue this PR will close: #311 

## Checklist

Please make sure you've completed the relevant tasks for this PR, out of the following list:

* [x] Code compiles correctly
* [x] Created/updated tests
* [x] Extended the documentation

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,16 +6,10 @@ This project welcomes contributions and suggestions. Most contributions require
 agree to a Contributor License Agreement (CLA) declaring that you have the right to,
 and actually do, grant us the rights to use your contribution.
 
-For details, visit https://cla.microsoft.com.
-
 When you submit a pull request, a CLA-bot will automatically determine whether you need
```
Comment (mukundansundar) [inline CONTRIBUTING.md:9]: remove this para also

Comment (amulyavarote) [reply CONTRIBUTING.md:9]: I dont see it removed in go sdk PR.

Comment (mukundansundar) [reply CONTRIBUTING.md:9]: It should be removed. As this refers to the CLA bot which will no longer be used.

Comment (amulyavarote) [reply CONTRIBUTING.md:9]: Sure. I will remove it.

**Review by mukundansundar — APPROVED** [review]

**Review by berndverst — APPROVED** [review]

**Pull request merged**